src/dotfile.ts

```diff
@@ -19,14 +19,14 @@ import { Credentials } from 'google-auth-library';
 import { OAuth2ClientOptions } from 'google-auth-library/build/src/auth/oauth2client';
 import stripBom = require('strip-bom');
 
-declare type dotf = (dirname: string, name: string) => {
+export declare type Dotf = (dirname: string, name: string) => {
```

Comment:
We should really finish `dotf` first:
https://github.com/grant/dotf/pull/8

Reply:
Indeed. This declaration is temporary. `Dotf` vs `dotf` is mainly to avoid ambiguity between type and object at this point.